recipe/meta.yaml

```
@@ -49,18 +45,84 @@ requirements:
     - pytorch
     - pytorch =*=cuda*
     - setuptools
-  run:
-    - einops
-    - python
-    - pytorch =*=cuda*
 
-test:
-  imports:
-    - flash_attn
-  commands:
-    - pip check
-  requires:
-    - pip
+outputs:
+  - name: flash-attn
+    requirements:
+      build:
+        - {{ compiler('cxx') }} # needed for DSO checker
+        - {{ stdlib('c') }}     # needed for DSO checker
+      host:
+        - cuda-version {{ cuda_compiler_version }}  # same cuda for host and build
+        - cuda-cudart-dev  # [(cuda_compiler_version or "").startswith("12")]
+        - python
+        - pytorch =*=cuda*
+      run:
+        - einops
```

Would move what was in the top-level `requirements/run` to the `flash-attn` split package. This ensures `flash-attn` has the same runtime requirements as before

```suggestion
      run:
        - einops
        - python
        - pytorch =*=cuda*
```